ibm_catalog.json

```diff
@@ -0,0 +1,74 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-key-protect",
+      "label": "Key Protect module",
+      "product_kind": "module",
+      "tags": [
+        "dev_ops",
+        "target_terraform",
+        "terraform",
+        "module"
+      ],
+      "keywords": [
+        "terraform",
+        "key protect",
+        "encryption",
+        "policy"
+      ],
+      "short_description": "Terraform module to create and work with IBM Key Protect",
+      "long_description": "Use this module to provision and configure an IBM Key Protect instance.\nYou can configure the following aspects of your instance:\n- [Key Protect instance](https://cloud.ibm.com/docs/key-protect?topic=key-protect-about)\n- [Rotation policy](https://cloud.ibm.com/docs/key-protect?topic=key-protect-set-rotation-policy&interface=ui)\n- [Dual authorization policy](https://cloud.ibm.com/docs/key-protect?topic=key-protect-manage-dual-auth)\n- [Metrics policy](https://cloud.ibm.com/docs/key-protect?topic=key-protect-manage-monitor-metrics)\n- [Key create and import access policy](https://cloud.ibm.com/docs/key-protect?topic=key-protect-manage-keyCreateImportAccess)\n#### Usage\n```hcl\nmodule \"key_protect_module\" {\n  source  = \"terraform-ibm-modules/key-protect/ibm\"\n  version = \"latest\" # Replace \"latest\" with a release version to lock into a specific release\n  key_protect_name  = \"my-key-protect-instance\"\n  resource_group_id = \"xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX\"\n  region            = \"us-south\"\n}```",
```

Comment:
- source needs to be updated to `https://cm.globalcatalog.cloud.ibm.com/api/v1-beta/offering/source?archive=tgz&kind=terraform&name=terraform-ibm-key-protect&version=X.X.X`
- Move this comment so it shows above the source: `# Replace "X.X.X" with a release version to lock into a specific release`
- Missing a `\n` at the end. The ``` should be on the last line:
![image](https://github.com/terraform-ibm-modules/terraform-ibm-key-protect/assets/29608224/ed9e15ab-362e-4216-acfe-032050b458e6)